DI-2497/describe_beds_for_old_panel_app_versions.py

```
@@ -0,0 +1,134 @@
+        project name
+    start : str (required)
+        start date to look for projects from
+    end: str (requered)
+        end date to look for projects until
+
+    Returns
+    -------
+    projects : list
+        list of DNAnexus projects
+    """
+    analyses=test_code.split(".")[1]
```

Ruff (0.15.2)

[error] Local variable `analyses` is assigned to but never used

Remove assignment to unused variable `analyses`

(F841)